### ISSUE TITLE
Add automated checking for new versions.

### ISSUE THREAD

**Issue by feikname** [piqueserver/piqueserver#45]
With an option to disable it, of course.

**Comment by feikname** [comment]
Work-in-progress branch for this feature: [`add-new-version-warning`](https://github.com/piqueserver/piqueserver/tree/add-new-version-warning).

**Comment by NotAFile** [comment]
The branch is on par with master, is there any progress with this yet?

**Comment by feikname** [comment]
@notafile There wasn't any progress on it yet.

However, there was a discussion in Slack and there seems to be a consensus in implementing it by checking new versions with the GitHub API and adding a option to disable this feature is essential.

Also, an command-line action to simply check for a newer version could be nice.

e.g.
```text
$ piqueserver --version --check-for-updates
```

**Comment by NotAFile** [comment]
I don't think that would be very useful. The whole point of the message is that people forget to upgrade, so the only real way would be an e.g. daily check for updates.

**Comment by feikname** [comment]
Only at startup or while the program is running too? IMO, almost no one checks the logs after the server has started unless something bad is happening.

**Comment by NotAFile** [comment]
Well, it would get sent to IRC etc. too.
